### ISSUE TITLE
Look in differences in OCIO/AMP reports

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2638]
Our compliance score score went down 1% but our violations dropped went from to 37,226 to 19,890. Look more into how the % is calculated and make issues for any new or remaining violations.
